lib/screens/welcome_screen/welcome_screen.dart

```diff
@@ -59,9 +59,9 @@ class _WelcomeScreenState extends State<WelcomeScreen> {
     )
   ];
   List<String> transferMessages = [
-    'Sending file ...',
-    'File sent',
-    'Oops! something went wrong'
+    TextStrings().selectFile,
```

Comment:
@Gayathri-GA are we replacing "sending file..." text with TextStrings().selectFile which is "Select a File" ?